codecov_auth/helpers.py

```diff
@@ -28,3 +33,35 @@ def current_user_part_of_org(owner, org):
     # owner is a direct member of the org
     orgs_of_user = owner.organizations or []
     return org.ownerid in orgs_of_user
+
+
+# https://stackoverflow.com/questions/7905106/adding-a-log-entry-for-an-action-by-a-user-in-a-django-ap
+add = ADDITION
+change = CHANGE
+delete = DELETION
+
+
+class History:
+    @staticmethod
+    def log(objects, message, user, action_flag=None, add_traceback=False):
```

Comment:
could you add a comment describing what these arguments are meant to be? particularly `objects` and `action_flag`.

looking at the tests, you are logging that `orig_owner` made a `change` related to `impersonated_user`, but that could also be a list of impersonated users? what are some other use cases of this that would want a list of objects?

Reply:
This helper is just being used in the case where we're impersonating users, but in the future we might use this helper to log other actions in the admin log, some of those actions may involve multiple objects instead of just one.